tests/instruments/hcp/test_tc038d.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+"""
+Unit tests for the HCP TC038D
+"""
+
+# IMPORTS #####################################################################
+
+
+import pytest
+
+from pymeasure.test import expected_protocol
+
+
+from pymeasure.instruments.hcp import TC038D
+
+
+def test_write_multiple():
+    # Communication from manual.
+    with expected_protocol(
+        TC038D,
+        [(b"\x01\x10\x01\x0A\x00\x04\x08\x00\x00\x03\xE8\xFF\xFF\xFC\x18\x8D\xE9",
+          b"\x01\x10\x01\x0A\x00\x04\xE0\x34")]
+    ) as inst:
+        inst.writeMultiple(0x010A, [1000, -1000])
+
+
+def test_write_multiple_CRC_error():
+    with expected_protocol(
+        TC038D,
+        [(b"\x01\x10\x01\x06\x00\x02\x04\x00\x00\x01A\xbf\xb5",
+          b"\x01\x10\x01\x06\x00\x02\x01\x02")],
+    ) as inst:
+        with pytest.raises(ConnectionError):
+            inst.setpoint = 32.1
+
+
+def test_write_multiple_wrong_values():
+    with expected_protocol(
+        TC038D, []
+    ) as inst:
+        with pytest.raises(ValueError):
+            inst.writeMultiple(0x010A, 5.5)
+
+
+def test_write_multiple_Value_error():
```

Comment:
```suggestion
def test_write_multiple_handle_wrong_start_address():
```
?

Reply:
Also, maybe those tests checking internal error responses (CRC, start address, etc below) can use a docstring explaining what exactly is being tested for, how and why?